plugins/destination/plugin.go

```diff
@@ -184,66 +187,62 @@ func (p *Plugin) Init(ctx context.Context, logger zerolog.Logger, spec specs.Des
 }
 
 // we implement all DestinationClient functions so we can hook into pre-post behavior
-func (p *Plugin) Migrate(ctx context.Context, tables schema.Tables) error {
-	SetDestinationManagedCqColumns(tables)
```

Comment:
So is the expectation for the internal columns to be set in source plugins? I think that's fine, just means will need to update all sources before releasing any destinations

Reply:
I think we already do that on the source side (if not we might want to keep it here but I believe we changed it long time a go and this is why I add an error if it doesn't happen)